generators/languages/templates/src/main/webapp/i18n/de/metrics.json

```diff
@@ -74,8 +74,16 @@
             "title": "Cache Statistiken",
             "cachename": "Cache Name",
             "hits": "Treffer",
-            "misses": "Keine Treffer",
-            "evictions": "Anzahl entfernter Objekte"
+            "misses": "Fehler",
+            "gets": "Anzahl gelesener Objekte",
+            "puts": "Anzahl geschriebener Objekte",
+            "removals": "Anzahl gelöschter Objekte",
+            "evictions": "Anzahl entfernter Objekte",
+            "hitPercent": "Trefferwahrscheinlichkeit %",
```

Comment:
As a native german speaker I think this is not totally correct. I think the metric describes the hit and miss ratio and not the hit or miss probability.  So maybe "(Cache)-Treffer %" and "(Cache)-Fehler %".

Reply:
A native German speaker in Munich did the translations; I ran the PR submission. 

The missing keys came up in a Gitter conversation. I'm not so invested in the debate of the translation as the missing keys. With the keys in place, a discussion of the translation is fair game @atomfrede .

Reply:
Yes the actual translation is not that important and easy to change. Thanks for the pr!

Reply:
Yes, you are right. I think it should be "(Cache)-Treffer %" or "Trefferquote %".

Reply:
Both seem fine. Cache-Treffer is close to to the english one, while Trefferquote would be more common german wording. I would go with trefferquote maybe.